website/repairnator-mongo-rest-api/server/tests/inspector.test.js

```diff
@@ -18,21 +18,4 @@ describe('## Inspector', () => {
         .catch(done);
     });
   });
-  describe('# GET /repairnator-mongo-api/inspectors/status/:status', () => {
```

Comment:
The replacement is the other PR #748 ?

Reply:
No this is not related to the other PR.

Reply:
But wait I realize something is missing.

Reply:
Okay. It just seemed odd to remove something completely without replacing it at all. I assume the same thing is part of something else, but saw no addition anywhere else.

Reply:
Thank you, I actually forgot that part of the modification were not included... 
But now we gonna have some merge conflict with the yarn.lock, this is because I updated mongoose in the same time on a different branch. 
I'll investigate it.

Reply:
Okay. Sounds good.

Reply:
Finally everything is alright !!! 
Yarn is a nightmare :(